xmtp_mls/src/builder.rs

```diff
@@ -529,7 +530,7 @@ mod tests {
             inbox_id: inbox_id.clone(),
             installation_keys: SignatureKeyPair::new(SignatureScheme::ED25519).unwrap(),
             credential: Credential::new(CredentialType::Basic, rand_vec()),
-            signature_request: None,
+            signature_request: Mutex::default(),
```

Comment:
Is there a way we can pull this out of `xmtp_mls`, and use a `Mutex` directly in `bindings_ffi` and `bindings_node`?

Reply:
Switching to an atomic to avoid potential deadlocks in the future.